src/renderer/services/deposit/networks.ts

```diff
@@ -0,0 +1,40 @@
+import {config as mainnetBeaconConfig} from "@chainsafe/eth2.0-config/lib/presets/mainnet";
+import {config as minimalBeaconConfig} from "@chainsafe/eth2.0-config/lib/presets/minimal";
+import {INetworkConfig} from "../interfaces";
+import {ethers} from "ethers";
+import {JsonRpcProvider} from "ethers/providers";
+
+const isLocal = process.env.NODE_ENV !== "production";
+
+const networks: INetworkConfig[] = [
+    {
+        networkName: "Dummy",
+        networkId: 0,
+        contract: {
+            address: "0x00000000000001",
+            depositAmount: 32,
```

Comment:
should this be in WEI? Prysm has 3.2 ETH for example so will be easier to handle decimals immeaditely

Reply:
Can't we just write 3.2? haha

Reply:
I guess it's supported by the current code